pages/common/enca.md

```diff
@@ -0,0 +1,19 @@
+# enca
+
+> Detect and convert encoding of text files
+
+- detect file encoding; -L option tell enca the current language; LANGUAGE as zh_CN, en_US etc.
```

Comment:
@joelhy only the "tell" --> "tells" was changed. Did you overlook the second suggestion, or do you disagree with the proposed wording?